frontend/public/components/checkbox.jsx

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-undef, no-unused-vars */
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+
+class Checkbox_ extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      items: {},
+      selectedOptions: props.selectedOptions,
+    };
+  }
+
+  onChange = key => {
```

Comment:
I went back and forth on this.  If I remove this code, keeping track of which items are selected falls to the calling class.  I thought this would be easier since people would not have to keep track of selectedItems.

Reply:
I think in most cases you'll only have one checkbox. When you have two, it's easy enough to have separate handlers for each.